File: TRIANGLE PATTERN 1/code6.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter your no. of rows : ");
	scanf("%d",&row);

	for(int i = 0; i < row ; i++){

		for(int j = 0 ; j <= i ;j++){

			if((i)%4== 0){

				printf("%d\t",j+1);

			}else if(i%2==1){


				
				printf("%c\t",65+ j);
			}else {

	
				printf("%c\t",97 + i);
			}

		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/7/code6.java
import java.util.Scanner;

class NestedForDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter no. of rows : ");
		int row = s.nextInt();

		for(int i = 1; i <= row ; i++){

			for(int j = 1; j <= row ; j++){

				System.out.print(j+"	");

			}
			System.out.println("\n");

		}

	}


}
